src/SmartComponents/CompletedTaskDetails/JobResultsDetails/ExpandedIssues.js

```diff
@@ -73,7 +73,11 @@ const ExpandedIssues = ({ rowPairs, isReportJson }) => {
             backgroundColor: '#ffffff',
           }}
         >
-          <CodeBlockCode>{rowPairs}</CodeBlockCode>
+          <CodeBlockCode>
+            {typeof rowPairs === 'object'
```

Comment:
We need to add 2 more conditions if we use `typeof`, [reference](https://bobbyhadz.com/blog/javascript-check-if-value-is-object#check-if-a-value-is-an-object-in-javascript). This one of the works confusing aspects of JS :)
```suggestion
            {typeof rowPairs === 'object' &&  rowPairs !== null &&
  !Array.isArray(rowPairs)
```

Reply:
Thanks @mkholjuraev.  If a tasks returns a null or an array then I think we'll get a similar Sentry error, but I guess we can cross that bridge when we come to it.